requirements/main.txt

```diff
@@ -1,4 +1,5 @@
 numpy>=1.15.0
 cma>=2.6.0
+smac>=0.13.1
```

Comment:
this is too small a requirements to go into main, please use bench.txt and import inline

Reply:
moved